workloads/scale-perf/run_scale_fromgit.sh

```diff
@@ -44,7 +44,7 @@ spec:
     name: scale_openshift
     args:
       label:
-        key: node-role.kubernetes.io/workload
+        key: node-role.kubernetes.io/${NODE_TO_SCALE:=worker}
```

Comment:
We usually set these variables in common.sh. Also I think is better to use WORKLOAD_NODE_ROLE rather than NODE_TO_SCALE

Reply:
yes makes sense , will do the needful